ui/src/main/java/edu/wpi/grip/ui/preview/PreviewsController.java

```diff
@@ -26,20 +29,20 @@
 @Singleton
 public class PreviewsController {
 
-    @FXML private HBox previewBox;
+    @FXML
```

Comment:
Then put a comment on those lines and I'll leave responses explaining the change. Or better yet, ask me to leave a comment explaining why I changed something.